main/common/common.go

```diff
@@ -38,10 +38,16 @@ var (
 	Logger     = flag.String("logger", "glog", "Selects the logger to use")
 )
 
+type UIConfig struct {
+	Port      int `yaml:"port"`
```

Comment:
there's two levels of configs.

1. running UI standalone (square/metrics) repo
2. running UI as part of an existing server (our internal repo).

these settings - port and timeout - are only required for (1) but the config was provided for both. i wanted to simplify the config from outside, thus split into two.